docs/design/alias-configuration.md

```diff
@@ -52,4 +69,56 @@ In this flow
 
 ## Open Questions
 
-## Acceptance Tests
\ No newline at end of file
+## Acceptance Tests
+
+## Cases
+
+- Crypto Transfer Transaction to EVM address alias should create an account with its alias field set to the EVM address and its key will be empty (to be populated by a signed transaction as part of lazy account create)
+- Crypto Transfer Transaction (without auto creation): to be called with EVM address (the account with this address is already auto lazy created)
```

Comment:
This can be any transaction not just crypto transfer.
For instance is my account was lazily created.
If i issued a ContractCall and signed from metamask it should be able to pick up my signature, fill in the account and carry on to process the ContractCall.
This should be seamless

Reply:
So we should have cases for `CryptoTransfer`, `ContractCreate`, `ContractCall`, `EthereumTransaction`, `TokenAssociation` to begin with.
We can discuss others but I believe these represent the next logical steps one might take

Reply:
Updated the description of this case to be for any type of signed transaction and listed `CryptoTransfer`, `ContractCreate`, `ContractCall`, `EthereumTransaction`, `TokenAssociation` as examples.